modules/torch/__init__.py

```diff
@@ -71,13 +92,17 @@ def update_screen(self, full=True):
             win.println("Left/right for")
             win.println("colour.")
             win.println()
+            win.println("B for flash mode. ")
+            win.println()
             
         win.println("LED: {}".format("ON" if self.state else "OFF"), 12)
         if self.led_h < 0:
             win.println("Colour: WHITE", 13)
         else:
             win.println("Colour: Hue={}'".format(int(self.led_h * 360)), 13)
         win.println("Brightness: {}%".format(int(self.led_v * 100)), 14)
+        win.println("Flash:{}  ".format(FLASH_PATTERNS[self.flash_mode][0]), 15)
+        win.println("Flash:{} of {}, {} ".format(self.flash_mode, len(FLASH_PATTERNS), self.flash_state), 16)
```

Comment:
Minor: can you make 0 show "Off" rather than "0 of 4"? And subtract 1 from len too I guess, as there are only actually 3 flash patterns?